src/components/evaluator/services/znkEvaluatorSrv.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+(function (angular) {
+    angular.module('znk.infra.evaluator').provider('ZnkEvaluatorSrv', function () {
+
+        var _evaluateQuestionGetter;
+
+        this.shouldEvaluateQuestion = function(evaluateQuestionGetter) {
```

Comment:
register function for evaluate.
changed the name to : shouldEvaluateQuestionFnGetter 
and param to: evaluateQuestionFn